test/UnitTests.proj

```diff
@@ -79,7 +79,7 @@
 
       <LockFiles Include="..\src\Tasks\Microsoft.NET.Build.Tasks.UnitTests\LockFiles\**\*.*" />
 
-      <AssetFiles Include="..\src\Assets\**\**\**\*.*" />
+      <AssetFiles Include="TestAssets\**\*.*" />
```

Comment:
Should we consider putting this behind a `$(TestAssets)` property?

Reply:
I think this is fine for now. The design of TestAssets is actually incorrect to begin with. It assumes that all test projects require *ALL* TestAssets, which is wrong. So, we do a lot of duplicate file copying. I'd have to refactor the test projects to make sense. I've discussed this with Marc, but it isn't something I can focus effort on right now.